### ISSUE TITLE
Translate into English?

### ISSUE THREAD

**Issue by akarelas** [selfteaching/the-craft-of-selfteaching#407]
Sounds like an interesting document, but sadly some people around the world know only English as a second language, instead of Chinese.

**Comment by xiaolai** [comment]
I do have a plan to give it localizations, including English; however, it might need three months.

**Comment by evolving-zd** [comment]
> I do have a plan to give it localizations, including English; however, it might need three months.

How about giving this chance to community?

**Comment by kingmodus** [comment]
How do we go about doing that

**Comment by xianrusso** [comment]
I would be keen on helping with any translation work if needed!

**Comment by devgrips** [comment]
> I do have a plan to give it localizations, including English; however, it might need three months.

I'm glad someone finally brought this up. I could help with Turkish and German if you ever need it after you translated to English.

**Comment by oxyf** [comment]
@xiaolai 
I could do Spanish once you have the English one, just let me know, I'd be more than glad to give a hand.

**Comment by duaneking** [comment]
Respectfully, it is now 2023. If the original author isn't going to get to this after years of these requests, I don't think it's going to happen.

I'm personally not going to learn Chinese to read this; The effort required to do that, And what I'm going to get in return for doing that, is simply not cost effective.

This text is also wildly out of date; A translation would be using old and out of date/bad data. I doubt even the setup would work given the versions used and the things that have changed in tech since then.

In short, You're probably better off considering this text to be a trap to waste your time with as its clearly been abandoned by its author.

**Comment by xiaolai** [comment]
感谢您的留言与支持。issue 区现作整理，本条关闭。书中内容如有错误或建议，欢迎提交 Pull Request。
